app/models/content_item.rb

```diff
@@ -33,9 +34,6 @@ def self.create_or_replace(base_path, attributes)
         type: "conflict",
         code: "409",
         message: e.message,
-        fields: {
-          transmitted_at: [e.message],
```

Comment:
Should this have payload_version as a field now? I assume it's not a "need" since it's survived this long with transmitted_at which could be wrong but just wondering if it's a nice to have?